### ISSUE TITLE
Uncaught (in promise) TypeError - Atom 1.17.0 x64, Windows 7

### ISSUE THREAD

**Issue by dqfan2012** [steelbrain/linter-ui-default#306]
I'm not sure if someone has already reported this error, but here it is. 

![panel_error](https://cloud.githubusercontent.com/assets/3586389/26158442/e2bee872-3ae9-11e7-9c31-1df6a6fc6240.png)

The lines in question

```js
  async refresh() {
    if (this.panel === null) {
      if (this.showPanelConfig) {
        await this.activate()
      } else return
    }
    const paneContainer = atom.workspace.paneContainerForItem(this.panel)
    if (!paneContainer && paneContainer === 'bottom') {
      return
    }
    if (
      (this.showPanelConfig) &&
      (!this.hidePanelWhenEmpty || this.showPanelStateMessages)
    ) {
      paneContainer.show()
    } else {
      paneContainer.hide()
    }
  }
```

**Comment by RachelScodes** [comment]
was just coming to report same issue.

**Comment by steelbrain** [comment]
Oh God, just found out the bug, `===` vs `!==` and `||` vs `&&`

Should be fixed in v1.5.1
